src/config/i18n/defaults.json

```diff
@@ -286,6 +286,7 @@
       "to": "To"
     },
     "sent_email_dialog": "Email Details",
+    "sent_email_summary": "Sent email to \"[[recipients]]\" with subject \"[[subject]]\"",
```

Comment:
so the structure of this arbitrary? like how subject is nested under sent_email?

Reply:
Dot notation is used to nest things where appropriate. Sent email has the to and subject labels in different places so have those nested, since this is a single blurb, opted for a single un-nested key.

Alternative would be something like this which I don't hate:

```
sent_mail
  summary
  dialog
     title
     to
     subject
```